Makefile

```diff
@@ -1,8 +1,11 @@
+CRATE_VERSION=$(shell head -4 Cargo.toml | grep ^version | awk '{print $$3}' | tr -d \")
```

Comment:
You can use the following command:

`cargo pkgid | cut -d# -f2 | cut -d: -f2`

Reply:
Nice. I didn't know about `cargo pkgid`.

I don't know my way around the `cut` command all that well, but for me, only part of this command was needed.

```
$ cargo pkgid | cut -d# -f2
0.1.0
```

Reply:
Yes, but the one I put is safer.

When the folder name and binary name are different, the pkgid will be like:
`file:///home/somebody/gitversion#binary_name:0.1.1`

To test it, just rename the folder name or the package name.